src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -162,4 +170,83 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses a string with multiple words from the find command, and check that it is not an empty string.
+     *
+     * @param searchString The search string.
+     * @return The trimmed search string.
+     * @throws ParseException If the search string is empty.
+     */
+    public static String parseMultipleWordsFromFindCommand(String searchString) throws ParseException {
+        requireNonNull(searchString);
+        String trimmedArgs = searchString.trim();
+        if (trimmedArgs.isEmpty()) {
+            throw new ParseException(
+                    String.format(MESSAGE_EMPTY_INPUT, FindCommand.MESSAGE_USAGE));
+        }
+        return trimmedArgs;
+    }
+
+    /**
+     * Parses a single word from the find command, and checks that it does not contain spaces.
+     *
+     * @param searchString The search string.
+     * @return The trimmed search string.
+     * @throws ParseException if the search string contains spaces.
+     */
+    public static String parseSingleWordFromFindCommand(String searchString) throws ParseException {
```

Comment:
This method seems to be trimming the searchString like parseMultipleWords. How do they differ?